src/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,133 @@
+import { FC, ReactNode, useState } from 'react';
+import * as RadixTooltip from '@radix-ui/react-tooltip';
+import tw, { css } from 'twin.macro';
+import { AnimatePresence, motion, Variants } from 'framer-motion';
+
+type TooltipProps = Omit<RadixTooltip.TooltipProps, 'open' | 'onOpenChange'> & {
+  /**
+   * *A single {@link JSX.Element}* as the handle to control the tooltip popover to be rendered where {@link Tooltip} is used.
+   *
+   * **Note:** A valid trigger here must be an HTML native element or an element that implement the {@link React.forwardRef} API.
+   */
+  trigger: ReactNode;
+  /**
+   * Theme color to be applied on the tooltip popover
+   */
+  theme?: 'dark' | 'light' | 'primary' | 'secondary';
```

Comment:
Appreciate the customizable, but I don't think adding `theme` to every component is necessary.

Reply:
Removed theme prop